plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/loadInterpreter.kt

```diff
@@ -6,49 +6,6 @@
 package org.jetbrains.kotlinx.dataframe.plugin
 
 import org.jetbrains.kotlin.fir.FirSession
-import org.jetbrains.kotlinx.dataframe.plugin.extensions.KotlinTypeFacade
```

Comment:
Can you revert this? It'll be easier for me to merge changes from DataFrame to Kotlin repo. I'll optimize imports there (and use start imports according to their codestyle)